simbev/main_simbev.py

```diff
@@ -23,16 +23,64 @@
 # SR_Metro - Metropole
 
 
-def run_simbev(region_ctr, region_id, region_data):
+def run_simbev(region_ctr, region_id, region_data, cfg, regions, tech_data, scenario_path, main_path):
     """Run simbev for single region"""
     print(f'===== Region: {region_id} ({region_ctr + 1}/{len(regions)}) =====')
 
+    # get timestep (in minutes)
+    stepsize = cfg.getint('basic', 'stepsize')
+
     # get probabilities
     probdata, wd = simbevMiD.get_prob(
         region_data.RegioStaR7,
         stepsize,
     )
 
+    # get days for simulation
+    days = cfg.getfloat('basic', 'days')
+
+    # get minimum soc value in %
+    soc_min = cfg.getfloat('basic', 'soc_min')
+
+    # read chargepoint probabilities
+    charge_prob_slow = pd.read_csv(os.path.join(scenario_path, cfg['charging_probabilities']['slow']))
+    charge_prob_slow = charge_prob_slow.set_index('destination')
+    charge_prob_fast = pd.read_csv(os.path.join(scenario_path, cfg['charging_probabilities']['fast']))
+    charge_prob_fast = charge_prob_fast.set_index('destination')
+
+    # set random seed from config or truly random if none is given
+    rng_seed = cfg['sim_params'].getint('seed', None)
+    rng = np.random.default_rng(rng_seed)
```

Comment:
The random initialization (both lines) **must** be done on init not in the thread! Otherwise each thread will start with the same random pool..
So please pass as arg instead..

Reply:
I fixed it now, thanks for catching this!